File: Medium/1980. Find Unique Binary String/1980. Find Unique Binary String.py
from typing import List

class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        result = ''

        for i in range(len(nums)):
            # If the current character at position i is '0', append '1'; otherwise, append '0'.
            result += '1' if nums[i][i] == '0' else '0'

        return result

File: Medium/1980. Find Unique Binary String/1980. Find Unique Binary String.rs
impl Solution {
    pub fn find_different_binary_string(nums: Vec<String>) -> String {
        let mut result = String::new();

        for i in 0..nums.len() {
            // If the current character at position i is '0', append '1'; otherwise, append '0'.
            result.push(if &nums[i][i..i + 1] == "0" { '1' } else { '0' });
        }

        result
    }
}
